.devcontainer/.my-microservice.yaml

```diff
@@ -0,0 +1,10 @@
+# Please adapt to package name:
+# rename the file name to match the
+# "prefix" configured in the config
+# module
+host: 127.0.0.1
+port: 8080
+log_level: info
+workers: 1
+auto_reload: true
+language: Greek
```

Comment:
The comment I had written got deleted, but I will reiterate. I take it you are hardcoding using Greek for the greeting language generation when you do an API call with this default config? I also imagine you can call the method and send this parameter with a different value?

Reply:
Yes exactly that would be possible, but it's just a toy example, so let's not put too much effort into it 😄